src/devtools/client/inspector/markup/new-markup.js

```diff
@@ -326,7 +326,7 @@ class MarkupView {
       this.expandNode(nodeId);
     }
 
-    this.store.dispatch(updateSelectedNode(nodeId));
+    this.selection.setNodeFront(this.nodes.get(nodeId));
```

Comment:
why do we not need to update the redux store?

Reply:
The selection will send a `new-node-front` event, which results in a call to `this.update()`, which will update the redux store.

Reply:
Nice